examples/multi/bundle/src/App.js

```diff
@@ -21,65 +21,58 @@
 
 import React, { Component } from "react";
 import "regenerator-runtime/runtime";
-
-import { login, getSession, logout, fetch } from "../../../../dist/index";
+import { SessionManager } from "../../../../dist/index";
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       status: "loading",
-      loginIssuer: "https://dev.inrupt.net",
+      loginIssuer: "https://oidc.dev.solid.nhs.janeirodigital.net",
```

Comment:
I assume this was a local temporary change, but it shouldn't be checked in, should it?

Reply:
Fixed in 018bd5248550384ad608f3ee55ea2b848a728a47